development/Dockerfile

```diff
@@ -28,7 +28,9 @@ ENV INVOKE_PYLINT_NAUTOBOT_LOCAL=true
 # Since this is only used for development and we don't ship this container, pinning Poetry back is not expressly necessary
 # We also don't need virtual environments in container
 RUN which poetry || curl -sSL https://install.python-poetry.org | python3 - && \
-  poetry config virtualenvs.create false
+  poetry config virtualenvs.create false && \
+  # Try to install the export plugin for Poetry v2, if it fails it's probably v1 so we can ignore it
+  poetry self add poetry-plugin-export || true
```

Comment:
If `NAUTOBOT_VER` is <= 2.4.17, it will be poetry v1. In the 2.4.18 and later docker tags, poetry will be v2.

Reply:
What do we need this export plugin for inside the container?